typescript/index.d.ts

```diff
@@ -74,7 +75,7 @@ declare namespace WoT {
      * WoT provides a unified representation for data exchange between Things, standardized in the Wot Things Description specification.
      * In this version of the API, Thing Descriptions is expected to be a parsed JSON object.
      */
-    export type ThingDescription = { [key: string]: any; };
+    export type ThingDescription = ThingDescriptionSchema;
```

Comment:
Note: We need to test but I recall some issues in node-wot if we change it object or so.
Hence we should check that before hands...